script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -1100,22 +1102,17 @@ She wouldn't be human if she were still alive after this.
 [sha:77b2d681ef46ce37927fdcf3c8e0b97602ef69ae]{
 -- Page 240, Offset 4639.
 -- なんでもなにもない。
-// Kinda a weird construction?
-// Is this supposed to be an expounded なんでもない -> I did it like it was nothing (kinda echoed in 4641)?
-// Or something about having no reason to do it?
-// First two lines also munged a bit for cleaner EN flow
-Did I really do this?
+But there's no question as to why.
 }
 [sha:867675ebdc35564048cbc9ed20b7a7660000944a]{
 -- Page 240, Offset 4640.
 -- たった今、自分の手で。
-Did %{i}I%{/i} really do this?
+Because just a few moments ago...
 }
 [sha:fc73677ee4ea07ef66f9932fb9e64e63a67de6b0]{
 -- Page 240, Offset 4641.
 -- 遠野志貴の手で、あっさりと、見知らぬ女をバラバラにしてしまったんじゃないか。
-Did I really just cut a woman I didn't even know to pieces,
- with my own hands, like it was nothing?
+Wasn't I the one who mutilated this stranger beyond recognition, and nobody else?
```

Comment:
The 'and nobody else' tagged on at the end feels very weak.

Reply:
I guess see other discussion